README.md

```diff
@@ -1,11 +1,3 @@
-Hypothesis client
-=================
-
-[![Build status](https://img.shields.io/travis/hypothesis/client/master.svg)][travis]
-[![npm version](https://img.shields.io/npm/v/hypothesis.svg)][npm]
-[![#hypothes.is IRC channel](https://img.shields.io/badge/IRC-%23hypothes.is-blue.svg)][irc]
-[![BSD licensed](https://img.shields.io/badge/license-BSD-blue.svg)][license]
-
```

Comment:
Ah - I presume this change was made locally while testing out orphan handling by editing the README, which is the test page that the server at http://localhost:3000 serves.